### ISSUE TITLE
ES6 module default export line trips linter

### ISSUE THREAD

**Issue by SeinopSys** [mozilla/addons-linter#2145]
### Describe the problem and steps to reproduce it:
I submitted an extension that contained a `connectivity.js` file (and others like it) with the following contents:

```js
const { BehaviorSubject } = rxjs;
const { distinctUntilChanged } = rxjs.operators;

const onlineSource = new BehaviorSubject(navigator.onLine);

class Connectivity {
	constructor() {
		this.online = onlineSource.asObservable().pipe(distinctUntilChanged());

		window.addEventListener('online', () => {
			onlineSource.next(true);
		});
		window.addEventListener('offline', () => {
			onlineSource.next(false);
		});
	}

	isOnline(){
		return onlineSource.value;
	}
}

export default (new Connectivity());
```

### What happened?
The linter blocked the zip file stating that there is a JS syntax error on the last line of said file.

### What did you expect to happen?
The extension should have passed validation without issues.

### Anything else we should know?
![Error with this.zip
100% complete · 94.10 KB of 94.10 KB
We have enabled a new linter to process your Add-on. Please make sure to report any issues on GitHub
Your add-on failed validation with 1 error.
JavaScript syntax error
See full validation report](https://user-images.githubusercontent.com/3200580/44076513-ba4bc17c-9fa0-11e8-9272-631c4e77efa0.png)
![JavaScript syntax error
Error: There is a JavaScript syntax error in your code; validation cannot continue on this file.
js/connectivity.js line 23 column 1](https://user-images.githubusercontent.com/3200580/44076550-dfc8c6ac-9fa0-11e8-9df6-a8d680fc099a.png)

I've verified that the extension works correctly on Firefox, all this blocking did is delay the update for Firefox users. While this new linter is active these should be skippable warnings in my opinion.

**Comment by EnTeQuAk** [comment]
Duplicate of https://github.com/mozilla/addons-linter/issues/2137

The fix will be heading to production this Thursday.
